Gems/AtomTressFX/External/Code/src/TressFX/TressFXAsset.cpp

```diff
@@ -776,7 +776,7 @@ namespace AMD
                 EI_Read((char*)&skinData.weight[j], sizeof(AMD::real32), ioObject);
             }
 
-            float weightSum = skinData.weight[0] + skinData.weight[1] + skinData.weight[2] + skinData.weight[3];
+            [[maybe_unused]] float weightSum = skinData.weight[0] + skinData.weight[1] + skinData.weight[2] + skinData.weight[3];
```

Comment:
since these are actual calculation that are not needed, I recommend to do:
```
#if defined(AZ_ENABLE_TRACING)
    float weightSum = skinData.weight[0] + skinData.weight[1] + skinData.weight[2] + skinData.weight[3];
#endif
```